test/e2e/api-specs/ConfirmationRejectionRule.ts

```diff
@@ -107,7 +107,6 @@ export class ConfirmationsRejectRule implements Rule {
         reject,
         task: async () => {
           try {
-            await this.driver.waitUntilXWindowHandles(3);
```

Comment:
should we remove this, `switchToWindowWithTitle` does wait now, but not sure how that works

Reply:
@HowardBraham any thoughts on if this is even needed? given that the fn call below waits for a window with matching title to appear now?